src/TreeSitter/Parser.hs

```diff
@@ -0,0 +1,17 @@
+module TreeSitter.Parser where
+
+import Prelude
```

Comment:
We don’t have `NoImplicitPrelude` on any more, so I think we don’t need the explicit import of `Prelude`.

Reply:
@robrix I'm going to add `tree-sitter-haskell` to `haskell-tree-sitter` today, so I'll clean up the `import Prelude`s in that PR.